src/ontology/mondo-edit.obo

```diff
@@ -22740,7 +22740,7 @@ replaced_by: MONDO:0016567
 
 [Term]
 id: MONDO:0001571
```

Comment:
could you please link the term tracker item to the terms you changed?

(I tried to do this myself, but when I open this branch, I don't see your changes, not sure what is going on).

We should add term tracker items to any changed terms going forward - it's a great way to track the rationale for our changes.

Reply:
done